### PR TITLE
[REF] do not request encryption if wallets were already created without it

### PR DESCRIPTION
**Pull request by gabrielbazan7**

### CODE REVIEW

File: src/providers/profile/profile.ts [hunk 1]
```diff
@@ -674,8 +674,14 @@ export class ProfileProvider {
 
   private askToEncryptKey(key): Promise<any> {
     if (!key) return Promise.resolve();
+    // if the key is already encrypted, keep it that way for new wallets
     if (key.isPrivKeyEncrypted()) return Promise.resolve();
 
+    // do not request encryption if wallets were already created without it
+    const wallets = this.getWallets({ keyId: key.id });
```
Comment (cmgustavo) [inline src/providers/profile/profile.ts]: At this point, you have already created the wallets.

Comment (cmgustavo) [reply src/providers/profile/profile.ts]: So, it will be true

Comment (gabrielbazan7) [reply src/providers/profile/profile.ts]: bindWalletClient function is called after the key encryption

Comment (cmgustavo) [reply src/providers/profile/profile.ts]: Ok, you're right.

**Pull request merged**